readme.md

```diff
@@ -1,5 +1,14 @@
-# OpenRCT2
-An open-source re-implementation of RollerCoaster Tycoon 2. A construction and management simulation video game that simulates amusement park management.
+
+
+<p align="center">
```

Comment:
`align="center"` is deprecated HTML. Use `style="text-align: center"` instead. (Same below)